.mailmap

```diff
@@ -1546,6 +1546,7 @@ fazledyn-or <ataf@openrefactory.com>
 foice <foice.news@gmail.com>
 goddus <39923708+goddus@users.noreply.github.com>
 gregmedlock <gmedlo@gmail.com>
+hacquees <ankitdiswar10@gmail.com>
```

Comment:
Is this how you want your name to be recorded in the AUTHORS file?

Reply:
Thanks, I changed it now.